extide/gradle/src/org/netbeans/modules/gradle/loaders/ExtensionPropertiesExtractor.java

```diff
@@ -170,7 +170,7 @@ public Collection<String> keys(BuildPropertiesSupport.Property container) {
             if (c == null) {
                 return null;
             }
-            return Arrays.asList(c.split(";;")).stream().map(s -> s.replaceAll("\\;", ";")).collect(Collectors.toList());
+            return Arrays.asList(c.split(";;"));
```

Comment:
Thanks for spotting this - actually the replacement should be from "\;" ("escape sequence") to ;, since list delimiter is set to ";;". Will fix in a separate PR.